### ISSUE TITLE
Feature request: add a 'sort by' functionality to the dropdown menu subreddit list 

### ISSUE THREAD

**Issue by HigherFive** [dxprog/reddit-booru#2]

**Comment by dxprog** [comment]
Going to pick your brain on what it is you want to sort by. You can have sort by subreddit, but is the secondary sort date? Implementing that as is would mean you'd have to go to the beginning of time to get to the next subreddit's images. There could be a date limit (last 24 hours, maybe). Sort by score would do the same unless some reddit-like ranking algo were implemented (or the above 24 hour limit).

Basically, my question is: how would you like to see this work?

**Comment by HigherFive** [comment]
Uuh, I think you are misunderstanding my request. I just wanted to see the subreddit **list** sorted. The one in the 'Show Posts From' dropdown menu. In retrospect, I could have worded that better.

**Comment by dxprog** [comment]
Well, I feel silly. Alphabetically, I presume?

**Comment by HigherFive** [comment]
That's what I was thinking of. When I said 'sort by' functionality I was imagining options like sort by popularity as well, but an alphabetical order is all I really want.

**Comment by dxprog** [comment]
4d91feb

**Comment by dxprog** [comment]
Fixed in commit 4d91feb

**Comment by HigherFive** [comment]
wohoo, yeah

**Comment by dxprog** [comment]
Still a few weeks out from launching (need to make sure mobile is correctly covered first), but here's a sneak peak. ![little sneak peak](http://i.snag.gy/hhjQk.jpg).
